src/protocolgame.cpp

```diff
@@ -2882,6 +2883,26 @@ void ProtocolGame::sendHouseWindow(uint32_t windowTextId, const std::string& tex
 	writeToOutputBuffer(msg);
 }
 
+void ProtocolGame::sendCombatAlalyzer(CombatType_t type, int32_t amount, DamageAnalyzerImpactType impactType, const std::string &target)
+{
+	NetworkMessage msg;
+	msg.addByte(0xCC);
+	if (impactType == DamageAnalyzerImpactType::HEALING) {
+		msg.addByte(HEALING);
+		msg.add<uint32_t>(amount);
+	} else if(impactType == DamageAnalyzerImpactType::RECEIVE) {
+		msg.addByte(RECEIVE);
+		msg.add<uint32_t>(amount);
+		msg.addByte(getClientElement(type));
+		msg.addString(target);
+	} else if (impactType == DamageAnalyzerImpactType::DEALT) {
+		msg.addByte(DEALT);
+		msg.add<uint32_t>(amount);
+		msg.addByte(getClientElement(type));
+	}
```

Comment:
You can move
```cpp
		msg.addByte(impactType);
		msg.add<uint32_t>(amount);
```
out of every branch because it will always be added.

Then, this should probably be a switch.